src/CoreWCF.BuildTools/src/AnalyzerReleases.Shipped.md

```diff
@@ -1,9 +1,17 @@
-﻿## Release 0.4
+﻿## Release 0.4.0
 
 ### New Rules
 
 Rule ID | Category | Severity | Notes
 --------|----------|----------|--------------------
 COREWCF_0100 | OperationParameterInjectionGenerator | Error | COREWCF_0100_OperationParameterInjectionGenerator
 COREWCF_0101 | OperationParameterInjectionGenerator | Error | COREWCF_0101_OperationParameterInjectionGenerator
-COREWCF_0102 | OperationParameterInjectionGenerator | Error | COREWCF_0102_OperationParameterInjectionGenerator
\ No newline at end of file
+COREWCF_0102 | OperationParameterInjectionGenerator | Error | COREWCF_0102_OperationParameterInjectionGenerator
+
+# Release 1.0.0
```

Comment:
@mconnew please update to what next version number will be

Reply:
As long as there's no new api, the next version will be 1.0.1

Reply:
the version number is updated